### PR TITLE
Roman Numeral Analysis add on (Reopen #220 after reverting merge)

### PR DESCRIPTION
**Pull request by neosatrapahereje**
I completely overlooked the discussion raised in issue #221, and merged #220 prematurely. 

This pull request reopens the merge. For more information, see the [description of the original pull request](https://github.com/CPJKU/partitura/pull/220#issue-1655900046)

### CODE REVIEW

**Comment by manoskary** [comment]
Still Needs Tests for RNA and ChordSymbols

File: partitura/io/importmusicxml.py [hunk 1]
```diff
@@ -435,6 +435,23 @@ def _parse_parts(document, part_dict):
         # when constructed from MusicXML
         score.set_end_times(part)
 
+        # Apply octave shifts directly to notes
+        for shift in part.iter_all(score.OctaveShiftDirection):
+            # Shifts normal notes
+            for note in part.iter_all(score.Note, start=shift.start.t, end=shift.end.t):
+                if note.staff == shift.staff:
+                    if shift.shift_type == "up":
```
[the remark this answers is not in the document]
Comment (neosatrapahereje) [reply partitura/io/importmusicxml.py:443]: Adding this comment again...

```python
# somewhere else (maybe in `utils.music`?)
OCTAVE_SHIFTS = {8: 1, 15: 2, 22:3}

# and then in the method
if shift.shift_type == "up":
    note.octave += OCTAVE_SHIFTS[shift.shift_size]
elif shift.shift_type == "down":
    note.octave -= OCTAVE_SHIFTS[shift.shift_size]
```

**Review by neosatrapahereje — COMMENTED** [review]
The changes look good (other than the comment about octave shift symbols)

**Review by neosatrapahereje — COMMENTED** [review]
Changes look fine.

**Pull request merged**